recipes-core/swupdate/files/acknowledge_update.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/sh
+#
+# Copyright (c) Siemens AG, 2021
+#
+# Authors:
+#  Quirin Gylstorff <quirin.gylstorff@siemens.com>
+#
+# SPDX-License-Identifier: MIT
+
+USTATE="$(fw_printenv -n ustate | tail -1)"
+
+if [ "$#" != "1" ]; then
```

Comment:
Why this check if you have a fallthrough in your case .. esac doing the same thing?